databroker_pack/_pack.py

```diff
@@ -199,12 +204,15 @@ def export_catalog(
       Resource documents, keyed on ``(root_in_document, root, unique_id)``.
     * ``failures`` is a list of uids of runs that raised Exceptions. (The
       relevant tracebacks are logged.)
+    * ``file_uids`` is a dictionary of RunStart unique IDs mapped to
+      ``(root, filename)``.
     """
     if limit is not None:
         if limit < 1:
             raise ValueError("limit must be None or a number 1 or greater")
         limit = int(limit)
     accumulated_files = collections.defaultdict(set)
+    file_uids = collections.defaultdict(list)
```

Comment:
Micro-optimization: we could simplify and avoiding paying for a shallow copy by making this a plain dict. Unlike with the other aggregations we are performing here, each pass through the loop will add one unique key (`uid`) to the dict. Subsequent passes through the loop will never need to append things to that same key.